### ISSUE TITLE
Add to /about page a section on the `Settings` values

### ISSUE THREAD

**Issue by rocky** [Mathics3/mathics-django#86]
In http://localhost/about it would be nice to include a section listing  describing what the current settings are.

You can get this with a combination of Mathics and Python. 

```
Names['Settings`*"] 
``` 

gives a list of setting names and in python, `to_python()` of the result will turn this into a Python list.

Then evaluating each of these like we did for `$QuotedStrings` in the code will give back the setting's value.

Even slicker would be to run ``??Settings`$<name>`` to get the docstring for that setting and show that.

There may be a way to do this as a one-liner in Mathics/WL, but I don't know enough about that to do this.

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky it have proved to don't be a too *good first issue*.

With the code bellow I'm seeing better by letter (literally) that the code didn't worked.
```py
def get_settings():
    from mathics.core.evaluation import Evaluation
    from mathics.core.expression import Expression

    settings = []

    obj = Evaluation(definitions, format='unformatted', output=WebOutput())
    names = Expression('Names["Settings`*"]').evaluate(obj).get_head().to_python()

    for setting in names:
        print (setting)

    return settings
```

Maybe a mistake in my `obj`?

**Comment by rocky** [comment]
@TiagoCavalcanteTrindade https://github.com/Mathics3/mathics-django/tree/add-settings-to-about has some hacky code to get the settings. Basically I used the function that Mathics `Names[]` uses, and then used some assumptions that there would be a single rule there and pulled the value from that. 

Consider this on par with the way I removed underscore between the logo and "Mathics' in a recent PR that you reviewed.

@mmatera probably would know a less hacky way to get the information.
